scripts/Manifest

```diff
@@ -11,7 +11,7 @@ KURL_BIN_UTILS_FILE=
 # STEP_VERSIONS array is generated by the server and injected at runtime based on supported k8s versions
 STEP_VERSIONS=(0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 1.16.4 1.17.13 1.18.20 1.19.16 1.20.15 1.21.14 1.22.17 1.23.17 1.24.17 1.25.14 1.26.15 1.27.16 1.28.15 1.29.14 1.30.10 1.31.6 1.32.2)
 # ROOK_STEP_VERSIONS array is generated by the server and injected at runtime based on supported rook versions
-ROOK_STEP_VERSIONS=(1.0.4-14.2.21 0.0.0 0.0.0 0.0.0 1.4.9 1.5.12 1.6.11 1.7.11 1.8.10 1.9.12 1.10.11 1.11.8 1.12.8)
+ROOK_STEP_VERSIONS=(1.0.4-14.2.21 0.0.0 0.0.0 0.0.0 1.4.9 1.5.12 1.6.11 1.7.11 1.8.10 1.9.12 1.10.11 1.11.8 1.12.8 1.13.10 1.14.12 1.15.8 1.16.3)
```

Comment:
I'm adding the 1.14/1.15/1.16 versions here so that it's easier to test upgrades to them in the dev work that will immediately follow this